resources/tests/org.eclipse.core.tests.resources/src/org/eclipse/core/tests/filesystem/FileStoreTest.java

```diff
@@ -90,7 +79,6 @@ public void testBrokenFetchInfo() {
 	private IFileStore getDirFileStore(String path) throws CoreException {
 		IFileStore store = EFS.getFileSystem(EFS.SCHEME_FILE).getStore(new Path(path));
 		store.mkdir(EFS.NONE, null);
-		deleteOnTearDown(store);
```

Comment:
why is that folder not removed anymore?

Reply:
but for example testCopyFileAcrossVolumes() does only deleteOnTearDown if and only if the test is not skipped!

Reply:
I agree. And even if the test is not skipped, the folder may still be created by the test and then has to be removed. See https://github.com/eclipse-platform/eclipse.platform/pull/441#discussion_r1187251273

Reply:
Good point, @jukzi, I missed that one. 
I'm going to take @HeikoKlare's  advice and add it for removal only if it didn't exist before.